Common/Securities/Cash.cs

```diff
@@ -251,6 +252,74 @@ SecurityChanges changes
                     Log.Trace("Cash.EnsureCurrencyDataFeed(): Adding " + symbol.Value + " for cash " + Symbol + " currency feed");
                     return security;
                 }
+
+
+            }
+
+
+            // No direct conversion rate pair is found, check for secondary conversion pair
+            // Common to cryptocurrencies where there are no direct pairings with USD, but there is intermediary such as BTC or ETH
+            // Example #1: RENUSD doesn't exist, but there is RENETH  and ETHUSD,  from which we can calculate RENUSD
+            // Example #2: RENUSD doesn't exist, but there is RENUSDT and USDTUSD, from which we can calculate RENUSD
```

Comment:
Looks like the code below (and this comment) are attempting to solve the indirect (multi-legged) conversion case. Please remove.